source/modules/MEI.header.xml

```diff
@@ -107,6 +107,62 @@
       </attDef>
     </attList>
   </classSpec>
+  <classSpec ident="att.perfRes" module="MEI.header" type="atts">
+    <desc>Attributes that define the characteristics and components of the performance resource.</desc>
+    <classes>
+      <memberOf key="att.perfRes.basic"/>
+      <memberOf key="att.transposition"/>
+    </classes>
+    <attList>
+      <attDef ident="solo" usage="opt">
+        <desc>Use this attribute to identify the performance resource as a soloist especially in an accompanied work, such as a concerto or vocal solo.</desc>
+        <datatype>
+          <rng:ref name="data.BOOLEAN"/>
+        </datatype>
+      </attDef>
+    </attList>
+  </classSpec>
+  <classSpec ident="att.perfRes.basic" module="MEI.header" type="atts">
+    <desc>Attributes that define the characteristics and components of the performance resource or a performance resource list.</desc>
+    <classes>
+      <memberOf key="att.adlibitum"/>
+    </classes>
+    <attList>
+      <attDef ident="count" usage="opt">
+        <desc>Indicates the number of performers.</desc>
+        <datatype>
+          <rng:data type="positiveInteger"/>
+        </datatype>
+      </attDef>
+    </attList>
+  </classSpec>
+  <classSpec ident="att.adlibitum" module="MEI.header" type="atts">
+    <desc>Attributes that define characteristics on an entity.</desc>
+    <attList>
+      <attDef ident="adlib" usage="opt">
+        <desc>Marks an entity as not strictly defined.</desc>
+        <datatype>
+          <rng:ref name="data.BOOLEAN"/>
+        </datatype>
+      </attDef>
+    </attList>
+    <remarks>
+      <p>The technical term “ad libitum” has several meanings depending on the context in which it occurs. It could</p>
```

Comment:
I think there is an unfinished sentence here.

Reply:
Thanks, might me a copy paste error.

Reply:
Ah, no, it belongs to the list following. But this solution was bad, I'll fix that!